core/includes/install.core.inc

```diff
@@ -1713,6 +1713,7 @@ function install_finished(&$install_state) {
   backdrop_cron_run();
 
   $message = st('Thank you for installing @profile!', array('@profile' => backdrop_install_profile_distribution_name()));
+  $message .= st(' If you are new to Backdrop, then <a href="@dashboard">the Dashboard</a> may be a good place to start.', array('@dashboard' => url('admin/dashboard')));
```

Comment:
In theory the install profile may not be the "standard" one, and the Dashboard module may not be enabled - in that case the link would be 404.

Reply:
I've added a profile check. Now trying to figure out how to test it.